### PR TITLE
docs: list eq in where filter

### PR DESCRIPTION
**Pull request by jannyHou**
This commit is fully migrated from [Ilia Smirnov](https://github.com/uniconstructor)'s PR in https://github.com/strongloop/loopback.io/pull/789. 

The original forked repository is unfortunately not found and therefore I could not edit the commit message or cherry pick the original commit. So I created this PR instead to merge Llia's doc contribution 🙇‍♀️ . Appreciate your time and effort on improving our project! especially for the very detailed usage description.

### CODE REVIEW

**Review by dhmlau — APPROVED** [review]
LGTM. @jannyHou, thanks for picking up this. Could you please look into the CI failure? Thanks.

File: pages/en/lb3/Where-filter.md [hunk 1]
```diff
@@ -185,7 +185,7 @@ This table describes the operators available in "where" filters. See [Examples]
 
 | Operator  | Description|
 | ------------- | ------------- |
-| = | Equivalence. See [examples](#equivalence) below.|
+| eq | Equivalence. See [examples](#equivalence) below.|
```
Comment (agnes512) [inline pages/en/lb3/Where-filter.md:188]: From my understanding, `=` maps to the `=` in `/weapons?filter[where][name]=M1911`. 

And I just tested on my end with LB4 app, I found that there are some issues with Node API and REST.

For Node, for filtering **numbers**, both 
`{where: {id: 1}}` and `{ where: {id:{'eq':1}} }` work.
But for filtering strings, none of them work:
`{where: {content: 'foooo'}}` and `{ where: {content:{'eq': "foooo"}} }`
( it returns all instances).


For REST, it has the same issue:
`http://localhost:3000/comments?filter[where][content]=foooo`returns all instances, while `http://localhost:3000/comments?filter[where][id]=1` works as expected.

**Comment by jannyHou** [comment]
@dhmlau I was trying to add Ilia Smirnov as the co-author, while his email is not public so I leave the email field empty, which fails the message parser. Fixed.

File: pages/en/lb3/Where-filter.md (hunk 1 above)
Comment (agnes512) [reply pages/en/lb3/Where-filter.md:188]: Sorry I didn't spot this before. I tested `where` with id so it seemed work to me :(

Comment (jannyHou) [reply pages/en/lb3/Where-filter.md:188]: @agnes512 Thank you for verifying it with a working app 👍 

> From my understanding, = maps to the = in /weapons?filter[where][name]=M1911.

Not really, `eq` here is one of the comparison operators among the list of [JS syntax operators](https://github.com/strongloop/loopback.io/blob/2d1a40d8b9c17a52c59120efd6dc2c0681c728a3/pages/en/lb3/Where-filter.md#operators). `=` is the [REST syntax](https://github.com/strongloop/loopback.io/blob/2d1a40d8b9c17a52c59120efd6dc2c0681c728a3/pages/en/lb3/Querying-data.md#rest-syntax) in url.

According to your observation, `eq` may have bug when the data is in type string, but based on my experience, it's the correct JS syntax, `=` is definitely not correct : p
We can create new issues to fix the problem you find. WDYT?

Comment (agnes512) [reply pages/en/lb3/Where-filter.md:188]: Agree, besides the bug, I think we should make it clear that `=` is for REST and `eq` for Node.

Comment (jannyHou) [reply pages/en/lb3/Where-filter.md:188]: @agnes512 cool. The rest syntax is documented here https://github.com/strongloop/loopback.io/blob/2d1a40d8b9c17a52c59120efd6dc2c0681c728a3/pages/en/lb3/Querying-data.md#rest-syntax
https://github.com/strongloop/loopback.io/pull/964/commits/e0d26a5eebea08b786d89edc2ed11ff379f4bc3f

**Review by agnes512 — APPROVED** [review]

**Comment by jannyHou** [comment]
Merging, all credit goes to @uniconstructor, thank you for the contribution again!

**Comment by agnes512** [comment]
Sorry that I made a mistake, when I was checking the syntax, I forgot to remove the hidden property from the model definition:
```
@model(
settings:{hidden:['content']}}
)
export class Comment extends Entity {
...
```

That's why it returned all instances as the filter was sanitized to `where:{}`. 

I re-test it again and the where clause works well 👍

**Comment by jannyHou** [comment]
@agnes512 🙇‍♀️ Thank you so much on the effort trying it! Good to know the new syntax also works  in LB4 💪

**Pull request merged**